modules/core/05-port/keeper/keeper.go

```diff
@@ -81,3 +81,8 @@ func (k *Keeper) LookupModuleByPort(ctx sdk.Context, portID string) (string, *ca
 
 	return types.GetModuleOwner(modules), capability, nil
 }
+
+// GetRoute() returns a IBCModule for a given module.
+func (k Keeper) GetRoute(module string) (types.IBCModule, bool) {
```

Comment:
I think we should make this a method on `*Keeper` instead of the struct since it's referencing a mutable type `Router`, and also to stay consistent with the other methods.

Reply:
Done!